server/auth_server.go

```diff
@@ -744,13 +745,21 @@ func GetConfig(accessToken string, listOnly bool) (model.AuthConfig, error) {
 	settings = append(settings, providerSetting)
 	settings = append(settings, providerNameSetting)
 	settings = append(settings, authServiceLogSetting)
+	settings = append(settings, apiAuthShibbolethRedirectWhitelistSetting)
```

Comment:
lets read the setting only if config.Provider == "shibbolethconfig" ; that way we save a DB lookup for other providers